src/app/views/App.tsx

```diff
@@ -339,10 +347,27 @@ class App extends Component<IAppProps, IAppState> {
     if (mobileScreen) {
       sidebarWidth = layout = 'col-xs-12 col-sm-12';
     }
-
+    if (permissionModeType && this.state.hideDialog) {
+      this.showDialog();
+    }
+    const teamsapp = <a href={"https://www.bing.com/?form=000010"}>Sample Explorer Teams app</a>;
+    //TODO: put in the url when we have this set up
+    const rsc = <a href={RSC_URL}> resource specific consent</a>;
     return (
       // @ts-ignore
       <ThemeContext.Provider value={this.props.appTheme}>
+        {!permissionModeType && <Dialog
+          hidden={this.state.hideDialog}
+          dialogContentProps={{
+            title: `${messages['Application Permissions']}`,
```

Comment:
```suggestion
            title: `${translateMessage('Application Permissions')}`,
```

Reply:
So I reviewed the code, and I am pretty sure that calling `translateMessage()` is the more current method to making sure a message is properly translated.

If you don't use `translateMessage`, what happens is that it shows as "undefined" or blank text when it is not defined for that language. If you do use it though, it will default the text.

Here's an example of the translateMessage changes on the zh-CN locale:
![Here's an example of the translateMessage changes on the zh-CN locale](https://user-images.githubusercontent.com/49354780/124477706-2c4cf180-dd59-11eb-81bd-c161b5ebc751.png)

You can still see in the background that some of the other variables we made in separate PRs are not updated for this yet. I'm going to go through and PR/suggest edits for those as part of my localization review ^^

Reply:
OHHH NO THE DREADED UNDEFINED. really good catch @AyuAi!!!

Reply:
Thanks Ayu! Your advice is helped me learn a lot about localization!!